src/test/playground-ide_test.ts

```diff
@@ -649,6 +650,63 @@ suite('playground-ide', () => {
     assert.equal(historyLengthAfter, historyLengthBefore);
   });
 
+  test('reloading preview does not create a new iframe element', async () => {
+    // NOTE: For some reason, the parent window's history only seems to be
```

Comment:
Shouldn't need the separate origin in this test case, that's only for testing the history issue.